doc/release_notes.rst

```diff
@@ -0,0 +1,30 @@
+.. _npmx_zephyr_release_notes:
+
+Release notes
+#############
+
+See the release notes for the information about specific npmx-zephyr releases.
+
+[0.6.0] - 20/06/2023
+---------------------
+
+Added
+~~~~~
+
+- Added basic repository structure.
+- Added README.md file.
+- Added samples:
```

Comment:
Please use either file names with their extensions and no uppercasing, or names only keeping the proper uppercasing (the one used in the left-side menu).